src/components/ArtObjectPageComponents/PanelDetails/index.jsx

```diff
@@ -195,30 +200,30 @@ class Image extends Component {
 
     // This indicates that there was an error with rendering the Zoom component
     const { didCatchFailure } = this.state;
-    const showZoomImageView = Boolean(!didCatchFailure && object.id && activeImageIndex === 0);
     const renditionsExist = renditions?.length > 0;
+    const showZoomImageView = Boolean(!didCatchFailure && object.id && !renditionsExist && activeImageIndex === 0);
 
     let additionalStyle = {};
     let imageUrlToRender = '';
 
     // If we encountered failure during rendering of the Zoom component, we'll hide it
     // Additionally, if the user clicked on an image from the rendition thumbnails, we'll render it instead
-    if (!didCatchFailure && activeImageIndex === 0) {
+    if (!didCatchFailure && !renditionsExist && activeImageIndex === 0) {
       additionalStyle = { ...additionalStyle, display: 'none' };
     };
 
-    // We'll render the image from the object itself 
-    if (activeImageIndex === 0) {
-      imageUrlToRender = object.imageUrlLarge
-    }  // Otherwise, one of the renditions was clicked. So we need to render that image
+    // We'll render the image from the object renditions itself 
+    if (renditionsExist) {
+      // We have to build the image URL from the asset information
+      const asset = renditions[activeImageIndex];
+      imageUrlToRender = `${ui.netxBaseURL}/api/file/asset/${asset.id}/${asset.fileName}`;
+    }  
```

Comment:
If renditions are found to exist for our object, we'll render those instead using the asset ID and file name, per NetX docs [here](https://developer.netx.net/#file-retrieval-methods)